src/UXClient/Components/ScatterPlot/ScatterPlot.ts

```diff
@@ -59,6 +59,7 @@ class ScatterPlot extends ChartComponent {
 
     ScatterPlot(){}
     public render(data: any, options: any, aggregateExpressionOptions: any, fromSlider: boolean = false) {
+        data = Utils.standardizeTSStrings(data);
```

Comment:
wondering if it's time to have super.render(data, options, aggregateExpressionOptions) and offload some of the render setup and initialization to ChartComponent